src/servers/ZoneServer2016/classes/gridcell.ts

```diff
@@ -1,8 +1,6 @@
-import { BaseEntity } from "../entities/baseentity";
-
 export class GridCell {
   position: Float32Array;
-  objects: Array<BaseEntity> = [];
+  objects: Array<any> = [];
```

Comment:
the `unknown `type would be more appropriate

Reply:
The classes are not assignable to unknown type

Reply:
Then we could have a custom type containing all the classes